src/component/parent/index.js

```diff
@@ -561,7 +561,11 @@ export class ParentComponent extends BaseComponent {
 
             [ POST_MESSAGE.CLOSE ](source, data) {
 
-                this.close();
+                this.iframe.className = 'xcomponent-lightbox bounceOutDown';
```

Comment:
Few points here:

1. Again would be a little hesitant to hard code `bounceOutDown` here. Maybe we can just have an `xcomponent-closing` class so users can define whatever animation they want under that?

2. `this.iframe` will not always be available. I'd recomment putting any custom `close` code in `drivers.js` then doing something like

```
if (RENDER_DRIVERS[this.context].close) {
    RENDER_DRIVERS[this.context].close.call(this);
}
```

I'd also recommend calling running this code ^ from the `close()` method rather than here.